clients/admin-ui/src/home/types.ts

```diff
@@ -1,13 +1,18 @@
-export type ModuleCard = {
-  backgroundColor: string;
+import { ScopeRegistry } from "~/types/api";
+
+type ModuleCard = {
   description: string;
-  descriptionColor: string;
-  hoverBorderColor: string;
   href: string;
   key: number;
   name: string;
-  nameColor: string;
   sortOrder: number;
   title: string;
-  titleColor: string;
+  color: string;
```

Comment:
the `color`s were all consistently derived from each other, so I changed it to just take one `color`